app/Http/Controllers/ProjectsController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Project;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
+
+class ProjectsController extends Controller
+{
+
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function index(Project $project)
+    {
+        $data = $project->getProjectlists();
```

Comment:
a)projectlist?
what is it?

b) maybe static, without instance of class Project (Project::getProjectLists());